backend/blueprints/spa_api/service_layers/player/play_style.py

```diff
@@ -40,7 +40,7 @@ def create_from_id(cls, id_: str, raw=False, rank=None, replay_ids=None, playlis
                                                                     rank=rank, replay_ids=replay_ids,
                                                                     playlist=playlist, win=win)
         except NoResultFound:
-            raise CalculatedError(404, f"User has no replays for the selected playlist: {playlist}")
+            raise NoReplaysForPlaylist(f"User has no replays for the selected playlist: {playlist}")
```

Comment:
message is duplicated here and in the init above. Would give "User has no replays for the selected playlist: User has no replays for the selected playlist: {playlist}"